credentialz/credentialz.proto

```diff
@@ -510,11 +510,24 @@ message UserPolicy {
 message AuthorizedUsersResponse {
 }
 
+// Hash type's
+enum HashType {
+  HASH_TYPE_UNSPECIFIED = 0;
+  HASH_TYPE_CRYPT_MD5 = 1;
+  HASH_TYPE_CRYPT_SHA_2_512 = 2;
+}
+
+message CryptoHash {
+  // Specifies the hash type used and value
+  HashType hashType = 1;
+  string hashValue = 2;
+}
```

Comment:
Is this enum to be used in other messages than the Password message? If not then please move it inside it.

Reply:
Addressed this